content/projects/oop/animals/part2/_index.md

```diff
@@ -88,6 +88,7 @@ Your directory structure should look like this:
 2. The `Dog` and `Cat` class methods should still remain as per part 1 of the project.
 3. Make sure the `makeAllSounds()`/`make_all_sounds()` method still works properly.
 4. See the pseudocode examples below for what kinds of things should be tested.
+5. Also add tests for your Home class. Make sure that it makes all the right noises and that you can't adopt the same pet twice.
```

Comment:
Since there is more than one Home class method that return different things (i.e: 1. adopt_pet -> returns number of pets currently adopted, 2. make_all_sounds -> returns sounds of currently adopted pets) something like this instead:
"Include tests for your Home class. Make sure its methods return the correct values and that you can't adopt the same pet twice."